modular_skyrat/master_files/code/modules/mob/living/carbon/human_helpers.dm

```diff
@@ -8,3 +8,13 @@
 		return span_deadsay("[t_He] [t_is] limp and unresponsive; they're still twitching on occasion, perhaps [p_they()] can still be saved..!")
 	else
 		return span_deadsay("[t_He] [t_is] limp and unresponsive; there are no signs of life and they've degraded beyond revival...")
+
+///copies over clothing preferences like underwear to another human
+/mob/living/carbon/human/proc/copy_clothing_prefs(mob/living/carbon/human/destination)
+	destination.underwear = underwear
+	destination.underwear_color = underwear_color
+	destination.undershirt = undershirt
+	destination.socks = socks
+	destination.bra = bra //SKYRAT EDIT ADDITION
+	destination.bra_color = bra_color //SKYRAT EDIT ADDITION
+	destination.jumpsuit_style = jumpsuit_style
```

Comment:
This is how you override a proc and add stuff basically after the rest is done executing. `..()` calls the parent proc (which in this case is the one with the `/proc` in it, but it isn't always the case, it just goes to the next definition it can find in the implementation tree, which can sometimes be at the same level- Poke me on Discord if you want a better explanation), whilst the `. =` makes it so it sets the return value of the proc to that returned by calling the parent, which in turns basically allows you to just extend just about any proc this way (so long as you don't need to insert something in the middle of said parent proc).
```suggestion
/mob/living/carbon/human/copy_clothing_prefs(mob/living/carbon/human/destination)
	. = ..()

	destination.bra = bra
	destination.bra_color = bra_color
```